app-emulation/playonlinux/files/playonlinux-4.4-wxpython4-fix.patch

```diff
@@ -0,0 +1,10 @@
+--- a/python/mainwindow.py	2020-05-17 07:05:33.000000000 -0700
++++ b/python/mainwindow.py	2021-01-03 05:09:52.295838555 -0700
+@@ -240,7 +240,6 @@
+ 
+         ## List game
+         self.list_game = wx.TreeCtrl(self, 105, style=wx.TR_HIDE_ROOT | wx.TR_FULL_ROW_HIGHLIGHT)
+-        self.list_game.SetSpacing(0)
```

Comment:
That can be easily done using a sed instead of a specific patch

`sed -i 's/        self.list_game.SetSpacing(0)//' -e python/mainwindow.py || die "Failed to sed"`

Reply:
Unfortunately GH does not like spaces so be sure to check for the correct numbers of them